panels/history/ha-panel-history.html

```diff
@@ -114,9 +132,12 @@
 
   attached: function () {
     this.datePicker = new window.Pikaday({
-      field: this.$.datePicker.inputElement,
+      field: document.createElement('input'),
+      trigger: this.$.datePicker.inputElement,
       onSelect: this.hass.entityHistoryActions.changeCurrentDate,
     });
+    // Set the initial datePicker date, without triggering onSelect handler.
+    this.datePicker.setDate(this.selectedDate, true);
```

Comment:
Originally on first load there isn't actually a date selected on the pikaday control. This fixes that.